src/adapters/platforms/lambda.rs

```diff
@@ -82,9 +82,13 @@ impl Platform for LambdaPlatform {
             .ok_or(miette!("Couldn't get ARN of deployed lambda"))?;
 
         self.arn = Some(function_arn);
-        self.arn
+        let arn = self
+            .arn
             .clone()
             .ok_or_else(|| miette!("No ARN returned from AWS"))
+            .unwrap();
```

Comment:
I don't think we want to unwrap if we can just return the error instead.
```suggestion
            ?;
```